src/pages/case-study/datron.tsx

```diff
@@ -0,0 +1,223 @@
+            <img
+              alt="Time icon"
+              className={ouCss.outcome__icon}
+              src="/img/pages/case-study/icons/time.svg"
+            />
+            Quick migration from InfluxDB
+          </p>
+        </div>
+      </section>
+
+      <section className={clsx(seCss.section, caCss.card)}>
+        <p className={caCss.card__title}>
+          <span className={caCss.card__quote}>&ldquo;</span>DATRON builds
+          high-speed industrial CNC machines which output high-frequency sensor
+          data requiring massive ingestion rates at the data layer.
+          <span className={caCss.card__quote}>&rdquo;</span>
+        </p>
+
+        <p className={caCss.card__subtitle}>
+          In this case study, we summarize the experience of DATRON evaluating a
+          database that can handle high-ingestion rates for operational
+          integrity and showcase the improvements they saw.
+        </p>
+      </section>
+
+      <section className={seCss.section}>
+        <div
+          className={clsx(
+            "markdown",
+            seCss["section--inner"],
+            seCss["section--column"],
+          )}
+        >
+          <h3>Why DATRON migrated from InfluxDB to QuestDB</h3>
+          <p className="font-size--large">
+            The CNC milling machines that DATRON builds must function reliably
+            and continuously for decades. Medical applications and the aerospace
+            sector require that the mechanical components are manufactured
+            precisely each time with exceptionally low tolerances.
+          </p>
+          <p className="font-size--large">
+            DATRON rigorously tests their installed assemblies in long-term
+            studies over several months, which detect unusual vibrations, wear,
+            and temperature deviations. The metrics collected are the
+            environmental temperature alongside the machine&apos;s spindle RPM,
+            while an accelerometer measures the spindle position in three axes.
+            An Arduino polls the spindle sensors at a frequency in the two-digit
+            kHz range, meaning that the testing phase generates multiple
+            terabytes of data. Storing these measurements requires a stable,
+            high-performance database that can write and query time series data
+            efficiently.
+          </p>
+          <p className="font-size--large">
+            Initially, DATRON was experimenting with InfluxDB as a datastore but
+            immediately encountered problems ingesting high-throughput
+            workloads. A recurring issue that surfaced was InfluxDB consuming
+            too much system RAM and an inability to keep up with ingestion
+            rates, causing frequent memory cache flushing. QuestDB was a capable
+            drop-in replacement in this scenario and allowed DATRON to do more
+            with less hardware, more efficiently.
```

Comment:
```suggestion
            with less hardware.
```
is more impressive and reads better.